gateway/src/api/latest.rs

```diff
@@ -851,6 +851,19 @@ impl ApiBuilder {
         self.router = self
             .router
             .route("/", get(get_status))
+            .route(
+                "/versions",
+                get(|| async {
+                    axum::Json(VersionInfo {
+                        gateway: env!("CARGO_PKG_VERSION").parse().unwrap(),
+                        // For now, these use the same version as gateway.
+                        // Only one version is officially compatible, but more are in reality.
+                        cargo_shuttle: env!("CARGO_PKG_VERSION").parse().unwrap(),
+                        deployer: env!("CARGO_PKG_VERSION").parse().unwrap(),
+                        runtime: env!("CARGO_PKG_VERSION").parse().unwrap(),
```

Comment:
This is not accurate if we don't release in lockstep. Maybe we can add a comment also about releasing in lockstep to link the compatible versions more easily?